community/contributing/docs/docs-style-guide.md

```diff
@@ -0,0 +1,290 @@
+
+|**Do** |**Do Not**|
+|:------|:---------|
+| Open the `rosco.yaml` file. | Open the rosco.yaml file.|
+| Go to the `/docs/tutorials` directory. | Go to the /docs/tutorials directory.|
+| Open the `/.hal/config` file. | Open the /.hal/config file.|
+
+### Lists
+Group items in a list that are related to each other. Use a numbered list for instructions that need to be completed in a specific order.
+
+ - End each item in a list with a period if one or more items in the list are complete sentences. For the sake of consistency, normally either all items or none should be complete sentences.
+ - Use the number one (1.) for ordered lists.
+ - Use (+), (* ), or (-) for unordered lists.
+ - Leave a blank line after each list.
+ - Indent nested lists with four spaces (for example, ⋅⋅⋅⋅).
+ - List items may consist of multiple paragraphs. Each subsequent paragraph in a list item must be indented by either four spaces or one tab.
+ - The first line of a code block should be indented four spaces
+
+For example, an ordered list with a code block looks like this in Markdown:
+~~~
+1. Do this
+1. Do this
+1. Do this
+1. Run these commands:
+
+    ```
+	  curl -O https://raw.githubusercontent.com/spinnaker/halyard/master/install/debian/InstallHalyard.sh
+	  sudo bash InstallHalyard.sh
+	```
+
+1. Do the next thing
+~~~
+
+The rendered output look like:
+
+1. Do this
+1. Do this
+1. Do this
+1. Run these commands:
+
+    ```
+	curl -O https://raw.githubusercontent.com/spinnaker/halyard/master/install/debian/InstallHalyard.sh
+	sudo bash InstallHalyard.sh
+	```
+
+1. Do the next thing
+
+## Inline code formatting
+
+### Use code style for inline code and commands
+
+Use meaningful variable names that have a context rather than  'foo','bar', and similar meaningless variable names.
+
+Use a single backtick (\`) to surround inline code in a Markdown document. In Markdown:
+~~~~~~~~~
+Run `hal deploy apply` to deploy Spinnaker.
+~~~~~~~~~
+
+Use triple backticks to enclose a code block. In Markdown:
+~~~~~~~~~
+```json
+{
+  "firstName": "John",
+  "lastName": "Smith",
+  "age": 25
+}
+```
+~~~~~~~~~
+
+Remove trailing spaces in all code blocks.
+
+### Don't include the command prompt in code snippets
+
+|**Do** |**Do Not**|
+|:------|:---------|
+hal deploy apply| $ hal deploy apply
+
+### Separate commands from output
+
+Verify that the Pod is running on your chosen node:
+
+    kubectl get pods --output=wide
+
+The output is similar to this:
+
+    NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+    nginx    1/1       Running   0          13s    10.200.0.4   worker0
+
+### Use normal style for string and integer field values
+
+For field values of type string or integer, use normal style without quotation marks.
+
+|**Do** |**Do Not**|
+|:------|:---------|
+Set the value of `enabled` to True. | Set the value of `enabled` to "True".
+Set the value of `image` to nginx:1.8. | Set the value of `image` to `nginx:1.8`.
```

Comment:
But code-font/backtics, right?

Set the value of `enabled` to `True`. 
Set the value of `image` to `nginx:1.8`.

Reply:
Excellent catch - updated